source/data-processing.rst

```diff
@@ -0,0 +1,53 @@
+.. _doc_routinator_data_processing:
+
+Data Processing
+===============
+
+Fetching
+--------
+
+There are two protocols in use to transport RPKI data: rsync and the RPKI
+Repository Delta Protocol (RRDP), which relies on HTTPS. RRDP was designed to be
+the successor to rsync. As almost all RPKI repositories currently support both
+protocols, Routinator will prefer RRDP if available. 
```

Comment:
Minor nit: RRDP isn't something anyone operating an rsync server can upgrade to, it's only relevant in the RPKI context, so it's not a general replacement for rsync. Perhaps say something like "RRDP was designed to be the successor to rsync in the RPKI" ?